src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/mappers/RespondentDetailsMapper.java

```diff
@@ -13,13 +13,13 @@ public PartyDetailsModel map(AsylumCase asylumCase, CaseDataToServiceHearingValu
 
         return PartyDetailsModel.builder()
             .partyID(caseDataMapper.getRespondentPartyId(asylumCase))
-            .partyType(PartyType.IND.getPartyType())
+            .partyType(PartyType.ORG.getPartyType())
             .partyRole("RESP")
-            .individualDetails(
-                IndividualDetailsModel.builder()
-                    .firstName(caseDataMapper.getRespondentName(asylumCase))
-                    .lastName("(Home Office)")
-                    .preferredHearingChannel(caseDataMapper.getHearingChannel(asylumCase))
+            .organisationDetails(
+                OrganisationDetailsModel.builder()
+                    .organisationType(PartyType.ORG.getPartyType())
+                    .name(caseDataMapper.getRespondentName(asylumCase))
+                    .cftOrganisationID(null)
```

Comment:
would it be possible to remove the organisation id rather than populating it as null?